src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -194,8 +194,9 @@ var ReactElementValidator = {
         false,
         'React.createElement: type should not be null, undefined, boolean, or ' +
           'number. It should be a string (for DOM elements) or a ReactClass ' +
-          '(for composite components).%s',
-        getDeclarationErrorAddendum()
+          '(for composite components).%s%s',
+        getDeclarationErrorAddendum(),
+        ReactComponentTreeHook.getCurrentStackAddendum()
```

Comment:
@briandeheus 

Have you addressed this comment?

>I think in my ideal world here, we would display in [unknown] (at file.js:123) when we have the location info (based on the element being constructed – this code can probably be reordered to call ReactElement.createElement first) and leave that line off and just show the parent if we don't (since in [unknown] is useless and we won't have any better display name since it isn't a string/function type).